app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -0,0 +1,143 @@
+    onClickToList: () -> Unit,
+    id: String,
+    viewModel: DetailViewModel = hiltViewModel()
+) {
+    DetailScreen(onClickToList = onClickToList, id = id)
+}
+
+@Composable
+fun DetailScreen(modifier: Modifier = Modifier, onClickToList: () -> Unit, id: String) {
+    var cardFace by remember { mutableStateOf(CardFace.FRONT) }
+
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .wrapContentSize()
+            .size(192.dp, 240.dp)
+    ) {
+        TestFlipCard(cardFace, onClickToList, id)
+    }
+
+    if (cardFace == CardFace.FRONT) {
+        Button(
+            onClick = {
+                cardFace = CardFace.BACK
+            }, modifier = modifier.wrapContentSize()
+        ) {
+            Text(text = "Back")
+        }
+    }
+}
+
+@Composable
+fun TestFlipCard(cardFace: CardFace, onClickToList: () -> Unit, id: String) {
+    var scale by remember { mutableStateOf(1f) }
+    var rotation by remember { mutableStateOf(1f) }
+
+    FlipCard(
+        rotation = rotation,
+        modifier = Modifier
+            .fillMaxWidth()
+            .aspectRatio(.7f)
+            .graphicsLayer(
+                scaleX = scale, scaleY = scale, cameraDistance = 12f
+            ),
+        front = {
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(Color.Blue),
+                contentAlignment = Alignment.Center,
+            ) {
+                Text(
+                    text = id,
+                )
+            }
+        },
+        back = {
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(Color.Blue),
+                contentAlignment = Alignment.Center,
+            ) {
+                Text(
+                    text = "대화주제",
+                )
+            }
+        },
+    )
+
+    if (cardFace == CardFace.FRONT) {
+        LaunchedEffect(key1 = Unit) {
+            coroutineScope {
+                launch {
+                    animate(
+                        initialValue = 1f, targetValue = 1.7f, animationSpec = tween(
+                            durationMillis = 1000, easing = FastOutSlowInEasing
+                        )
+                    ) { value: Float, _: Float ->
+                        scale = value
+                    }
+
+                    animate(
+                        initialValue = 0f, targetValue = 180f, animationSpec = tween(
+                            durationMillis = 540, easing = FastOutSlowInEasing
+                        )
+                    ) { value, _ ->
+                        rotation = value
+                    }
+                }
+            }
+        }
+    } else {
+        LaunchedEffect(key1 = Unit) {
+            coroutineScope {
+                launch {
+                    animate(
+                        initialValue = 180f, targetValue = 0f, animationSpec = tween(
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)